lessons/Early-Math/commutative-property/langs/en-us/pages/claire-and-walker.adoc

```diff
@@ -0,0 +1,50 @@
+= Which Circle is Correct?
+
+++++
+<style>
+div.circleevalsexp { width: auto; }
+</style>
+++++
+For each of the expressions in words, look at the Circles that Claire and Walker drew. Then, choose which Circle is correct: Claire's, Walker's, or both? Justify your response.
+
+[.FillVerticalSpace,cols=".^1a,^.^4a,^.^6a,^.^6a,^.^10a", stripes="none", options="header"]
+|===
+|	 | Expression in words: | Claire's Circle: | Walker's Circle: | Who is correct? Justify.
+
+| @n
+| Find the quotient of 15 and 5. Multiply it by 6.
+| @show{(coe '(* (/ 15 5) 6))}
+| @show{(coe '(* (/ 5 15) 6))}
+| @ifsoln{Claire's Circle is correct. "The quotient of 15 and 5" translates to @math{15 \div 5}. Division is not Commutative.}
+
+| @n
+| Double 8. Now add 7.
+| @show{(coe '(+ (* 2 8) 7))}
+| @show{(coe '(+ 7 (* 2 8)))}
+| @ifsoln{Both Circles represent the expression in words. Claire's Circle more accurately reflects the stated order, however.}
+
+| @n
+| 5 less than the product of 5 and 20.
+| @show{(coe '(- 5 (* 5 20)))}
+| @show{(coe '(- (* 20 5) 5))}
+| @ifsoln{Walker's Circle is correct. Taking "5 less than" entails subtracting 5. Subtraction is not commutative.}
+
+| @n
+| One half of the quotient of 36 and 9.
+| @show{(coe '(* 1/2 (/ 36 9)))}
+| @show{(coe '(* (/ 36 9) 1/2))}
+| @ifsoln{Both Circles are correct. The Commutative Property suggests we can multiply two values in any order.}
+
+| @n
+| Subtract 6 from the 20 tripled.
+| @show{(coe '(- (* 20 3) 6))}
+| @show{(coe '(- 6 (* 3 20)))}
+| @ifsoln{Claire's Circle is correct. Subtraction is not commutative.}
+
+
+| @n
+| The product of 4 and the difference of 3 and 1.
+| @show{(coe '(* 4 (- 1 3)))}
+| @show{(coe '(* 4 (- 3 1)))}
+| @ifsoln{Walker's Circle is correct. Subtraction is not commutative; "the difference of 3 and 1" must be translated as @math{3 - 1}.}
+|===
```

Comment:
How about including an example that reinforces that division is not commutative?